This is to create mobile skeleton of my Microverse portfolio

thenarain opened the pull request:
Add files:

1. index.html
2. styles.css
3. Linters.yml
4. .gitignore
5. MIT.md
6. README
7. Icons and Images for portfolio

rotimiazeez reviewed (changes requested):
# Hi @thenarain 👋, 
## Good job so far!👏
### To highlight🔥
- No linter errors✔
- Good PR description👍
- Descriptive README👌
-  Good clean code😍
![](https://media.giphy.com/media/YVpIaYgJ3lpMk/giphy.gif)
#### You are a step close to the perfect user interface(UI) Kindly make the suggested change(s) and you'll be unstoppable🚀💯

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

thenarain commented on index.html:
```diff
@@ -0,0 +1,44 @@
+</head>
+
+<body>
+  <main class="main-block">
+    <header class="head-menu">
+      <a class="logo" href="#">My Logo</a>
+      <nav>
+        <img class="ham-menu" src="./Images/ham-Icon.png" alt="menu">
+      </nav>
+    </header>
+    <section class="home-page">
+      <h1 class="intro">I’m Kartikay
+        Glad to see you!</h1>
+      <p class="intro-description">I’m a software developer! I can help you
+        build a product, feature or website. Look through some of my work and
+        experience! If you like what you see and
+        have a project you need coded, don’t hesitate to contact me.</p>
+      <h3 class="bottom-connect">LET’S CONNECT</h3>
+      <ul class="icon-container">
+        <li><a class="social-icon" href="#"><img src="./Images/Twitter-icon.png" alt="Twitter"></a></li>
+        <li><a class="social-icon" href="#"><img src="./Images/Linkedin-icon.png" alt="Linkedin"></a></li>
+        <li><a class="social-icon" href="#"><img src="./Images/Medium-icon.png" alt="Medium"></a></li>
+        <li><a class="social-icon" href="#"><img src="./Images/Github-icon.png" alt="Github"></a></li>
+        <li><a class="social-icon" href="#"><img src="./Images/Angellist-icon.png" alt="Angellist"></a></li>
+      </ul>
+    </section>
+  </main>
```
![Screenshot from 2021-12-03 13-27-25](https://user-images.githubusercontent.com/52527576/144603356-bd69a1bf-5c0e-469c-8550-95ed9c4a4de4.png)

I have fixed the design.

VanessaAoki reviewed (approved):
## STATUS: APPROVED 🟢 
Hi @thenarain, 

Great job implementing all the changes required by the previous Code Reviewer, your project is complete! There is nothing else to say other than... it's time to merge it! 
Congratulations! 🎉 

<img alt="approved" src="https://64.media.tumblr.com/87e0b6469c957bf896cdec1cab646acf/tumblr_inline_pk1bdakCCp1rpcr9a_500.gif">

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Your README file looks very professional and the PR has a descriptive title/summary of the implemented changes :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 


Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

thenarain committed:
Add files and Images
Markup for portfolio
Styles for markup
.gitignore to ignore node_modules
setup github Actions
images for the project

thenarain committed:
Add Readme file and MIT license file

medaminedev66 commented on index.html:
```diff
@@ -0,0 +1,55 @@
+    <section class="home-page">
+      <div class="home-page-wrapper">
+        <h1 class="intro height">I’m Kartikay Glad to see you!</h1>
+        <!-- <h1 class="intro">Glad to see you!</h1> -->
```
[OPTIONAL] Kindly remove the commented code to keep your code clean.

medaminedev66 reviewed (approved):
## Hi @thenarain, 

# Highlights

- The repo looks professional ✔️
- Git/GitHub flow best practices have been followed ✔️
- There are no linter errors ✔️
- You have a descriptive PR ✔️

# Status: Approved ✔️
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢
Congratulations! 🎉 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
